### DISCUSSION TITLE
How to get the homepage like the demo/DDEV with manual install ?

### DISCUSSION THREAD

**Discussion by unes** [GetDKAN/dkan#4161] [Q&A]
Hello, I installed DKAN latest with `composer...recommend-project` and installed the frontend following manual setup in `GetDKAN/data-catalog-app` but can't figure out how to fully activate the decoupled front end, make the front page like the demo (I imported the 10 basic sample data), the main domain name shows the basic Drupal theme with tags not clickable. Also is there a guide/tutorial on how to use the Drupal theming for showing datasets ?

**Comment by unes** [comment]
now the homepage is working, added `/home` in `admin/config/system/site-information` but what about Drupal native theming, and authentication on the JS front end?

**Comment by dafeder** [comment]
@unes the JS frontend on the demo site is designed to just present the public-facing pages and does not include any authentication logic or integration.

We are actually very close to adding much more support for handling dataset pages and search with standard Drupal theming - please stay tuned.

**Reply by unes** [reply]
Thanks @dafeder ! Looking forward to it.
